veidt/models.py

```diff
@@ -68,8 +70,23 @@ def predict(self, inputs):
         scaled_descriptors = self.preprocessor.transform(descriptors)
         return self.model.predict(scaled_descriptors)
 
-    def save(self, model_fname):
+    def model_save(self, model_fname, scaler_fname):
```

Comment:
Why do you rename the method as model_save? What does save not accomplish, given that it is already a model object? Also, if you really want to be verbose, isn't save_model the grammatically correct version?